draft-tiesel-v6ops-ipv6-app-testing.md

```diff
@@ -82,7 +96,10 @@ and explains common regressions to avoid when deploying IPv6 support.
 # Introduction
 
 For the last 20 years, enabling applications for IPv6 has focused on coexistence with IPv4 and allowing traffic to shift towards IPv6 without breaking IPv4 operation.
-With the US mandate to move all governmental agencies to "IPv6-only" [M-21-07], this target for IPv6 support changed to being fully functional in the absence of IPv4 and transition technologies providing connectivity to the IPv4 Internet.
+Motivated by a series of national regulations mandating state entities to proceed in the migration to IPv6, e.g., in
```

Comment:
```suggestion
This target has changed in part due to a series of national regulations mandating state entities to proceed in the migration to IPv6, e.g., in
```

Swapping the sentence structure made this easier to understand for me, YMMV.

Sorry for the three part edit. Making multiline suggestions on a touchscreen is interesting.